includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -263,6 +263,19 @@ public function create_intention(
 		$request['payment_method_types'] = $payment_methods;
 		$request['capture_method']       = $capture_method;
 
+		if ( $this->is_fraud_prevention_enabled_for_store() ) {
+			$request['metadata']['fingerprinting_avaialable'] = true;
```

Comment:
I suggest to replace `fingerprint` with `fraud_prevention` here and further to make the purpose of data collection clear to customers.

Reply:
I agree, I think that makes more sense. I'll work on it